docs/modules/user-guide/partials/proc_refering-to-a-parent-devfile-in-a-devfile.adoc

```diff
@@ -2,17 +2,22 @@
 = Refering to a parent devfile in a devfile
```

Comment:
Not part of this PR, but wanted to note that "referring" is spelled incorrectly. :)